src/module.c

```diff
@@ -2925,17 +3000,17 @@ static int MastersFanoutCommandHandler(RedisModuleCtx *ctx,
 
 static int FanoutCommandHandlerWithIndexAtFirstArg(RedisModuleCtx *ctx,
   RedisModuleString **argv, int argc) {
-  MastersFanoutCommandHandler(ctx, argv, argc, 1);
+  return MastersFanoutCommandHandler(ctx, argv, argc, 1);
```

Comment:
Was this a bug? Do we use the return value?

Reply:
Just raised a compilation warning, I don't think this is used somewhere.

Reply:
Checked, this value is actually ignored in redis. Command handler invoked at `cp->func(&ctx,(void**)c->argv,c->argc);`, in `RedisModuleCommandDispatcher()`.